index.css

```diff
@@ -0,0 +1,114 @@
+.a11yHidden, legend{
+  overflow:hidden;
+  position:absolute !important;
+  clip:rect(0, 0, 0, 0);
+  -webkit-clip-path:inset(50%);
+  clip-path:inset(50%);
+  width:1px;
+  height:1px;
+  margin:-1px
+}
+/* MAIN */
+.container{
+
+}
+main{
```

Comment:
what is the difference between `main` and `.main` selector?
why and when would you use either one?